# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/Histograms.json
# Commit: 99e71b905661
# Full Hash: 99e71b9056618a081f8bc2fb7803ba208db3a5b0
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-10 09:45:41
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin,devtools-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 68f119112940 -r 99e71b905661 toolkit/components/telemetry/Histograms.json
--- a/toolkit/components/telemetry/Histograms.json	Tue Apr 09 15:26:45 2024 +0000
+++ b/toolkit/components/telemetry/Histograms.json	Tue Apr 09 15:26:46 2024 +0000
@@ -4815,7 +4815,7 @@
     "expires_in_version": "never",
     "kind": "enumerated",
     "n_values": 6,
-    "description": "encoding removed: 0=unknown, 1=gzip, 2=deflate, 3=brotli"
+    "description": "encoding removed: 0=unknown, 1=gzip, 2=deflate, 3=brotli, 4=zstd"
   },
   "CACHE_LM_INCONSISTENT": {
     "record_in_processes": ["main", "content"],
